exporter/signalfxexporter/README.md

```diff
@@ -74,6 +74,9 @@ The following configuration options can also be configured:
   processor is enabled in the pipeline with one of the cloud provider detectors
   or environment variable detector setting a unique value to `host.name` attribute
   within your k8s cluster. And keep `override=true` in resourcedetection config.
+- `allowed_dimension_chars`: (default = `"_-"`) A string of characters 
```

Comment:
~`allowed_nonalphanumeric_chars` to be more specific?

Reply:
Yeah, that is more explicit, thanks for the suggestion.